janusgraph-server/src/main/java/org/janusgraph/graphdb/grpc/schema/SchemaManagerProvider.java

```diff
@@ -0,0 +1,61 @@
+// Copyright 2021 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.graphdb.grpc.schema;
+
+import org.janusgraph.core.JanusGraph;
+import org.janusgraph.core.schema.JanusGraphManagement;
+import org.janusgraph.core.schema.VertexLabelMaker;
+import org.janusgraph.graphdb.grpc.types.VertexLabel;
+
+public class SchemaManagerProvider {
+    private final JanusGraphManagement management;
+
+    public SchemaManagerProvider(JanusGraph graph) {
+        this.management = graph.openManagement();
+    }
+
+    //TODO: only for testing purpose at the moment.
+    public void createVertexLabel(VertexLabel vertexLabel) {
+        VertexLabelMaker vertexLabelMaker = management.makeVertexLabel(vertexLabel.getName());
+        if (vertexLabel.getReadOnly()) {
+            vertexLabelMaker.setStatic();
+        }
+        if (vertexLabel.getPartitioned()) {
+            vertexLabelMaker.partition();
+        }
+        vertexLabelMaker.make();
+
+        management.commit();
```

Comment:
It makes sense to commit after creating a new vertex label, however, committing at the end of this method means this SchemaManagerProvider object can only be used once, i.e. one cannot "getVertexLabelByName" after "createVertexLabel" because the management handler is already closed (committed).

I am not sure what the best approach is. Do we want to give SchemaManagerProvider the control of JanusGraphManagement's lifecycle, or give the control to the RPC service?

Reply:
I think we have find a solution to manage the lifecycle. 

I justed add this method for test reason (I will check to move this method into the test package). My plan is first add explore features for the schema and than start with adding and removing.

Reply:
Fair enough.